src/main/java/de/tum/in/www1/artemis/service/scheduled/quiz/QuizScheduleService.java

```diff
@@ -559,7 +557,7 @@ private void removeUnnecessaryObjectsBeforeSendingToClient(StudentParticipation
      * @param userSubmissionMap a Map with all submissions for the given quizExercise mapped by the username
      * @param userBatchMap      a Map of the username to quiz batch id for the given quizExercise
      * @param batchCache        a Map of all the batches for the given quizExercise
-     * @return the number of processed submissions (submit or timeout)
+     * @return                  the number of processed submissions (submit or timeout)
```

Comment:
I don't understand what you have done here. It seems that you have added unnecessary whitespace

Reply:
Every description (of the parameters and the return value) is now on the same vertical line.